pkg/generator/gen_main_test.go

```diff
@@ -24,23 +24,46 @@ const mainExp = `package main
 import (
 	"context"
 	"runtime"
+	"net/http"
 
 	stub "github.com/example-inc/app-operator/pkg/stub"
 	sdk "github.com/operator-framework/operator-sdk/pkg/sdk"
 	k8sutil "github.com/operator-framework/operator-sdk/pkg/util/k8sutil"
 	sdkVersion "github.com/operator-framework/operator-sdk/version"
 
+	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"github.com/sirupsen/logrus"
 )
 
+// Prometheus metrics port
+const promPort = ":9090"
```

Comment:
Can we use a less common port number at a higher range like 90000.
Also can you define the port number as a global constant somewhere other than the test file like in [k8sutil/constants.go](https://github.com/operator-framework/operator-sdk/blob/master/pkg/util/k8sutil/constants.go). That way we can use it everywhere in the generator package without redefining it.

Reply:
Or something that's actually in the ephemeral port range like 60K. 90K is out of range.